sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -98,7 +98,8 @@
 
 TableFieldSchema: Describes the schema (type, name) for one field.
   Has several attributes, including 'name' and 'type'. Common values for
-  the type attribute are: 'STRING', 'INTEGER', 'FLOAT', 'BOOLEAN', 'NUMERIC'.
+  the type attribute are: 'STRING', 'INTEGER', 'FLOAT', 'BOOLEAN', 'NUMERIC',
+  'GEOGRAPHY'.
```

Comment:
Add documentation on what should be written when writing GEOGRAPHY values ? And also what to expect when reading GEOGRAPHY values ?

Reply:
Done.